other/Dockerfile

```diff
@@ -40,11 +48,7 @@ ADD prisma .
 RUN npx prisma generate
 
 ADD . .
-
-# Mount the secret and set it as an environment variable and run the build
-RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
-    export SENTRY_AUTH_TOKEN=$(cat /run/secrets/SENTRY_AUTH_TOKEN) && \
-    npm run build
```

Comment:
This should not be changed. Learn more about why it's accessed in this way from this PR: https://github.com/epicweb-dev/epic-stack/pull/660/files#diff-68a2c4544d4339258d27d291ed844a8fc6a79ad1b1cc7dcce28594c8d01c12beR45

Reply:
Im not sure I can help then. When I add that back I'm back to the same errors

```
#17 30.85 [sentry-vite-plugin] Warning: No organization slug provided. Will not create release. Please set the `org` option to your Sentry organization slug.
#17 30.85 [sentry-vite-plugin] Warning: No org provided. Will not upload source maps. Please set the `org` option to your Sentry organization slug.
#17 30.93 [sentry-vite-plugin] Warning: No organization slug provided. Will not create release. Please set the `org` option to your Sentry organization slug.
#17 30.93 [sentry-vite-plugin] Warning: No org provided. Will not upload source maps. Please set the `org` option to your Sentry organization slug.
```

Reply:
Did you make sure to set those values in GitHub actions as secrets?

I did the same to my own app just now and it worked well :)

https://github.com/kentcdodds/kentcdodds.com/commit/538080f2055a5090e904634f31259088dc8d603c

In my case, I just hard-coded the org and project though so maybe that's why?

Reply:
I followed your commit and it works for me too. No errors and source maps generated successfully. I have updated this PR.